clusters/.terraform.lock.hcl

```diff
@@ -41,6 +41,7 @@ provider "registry.terraform.io/rancher/rancher2" {
   version = "1.21.0"
   hashes = [
     "h1:8VYSDYxLlXm2Do3sPVwlk6NCErwbmsH0FykHDh4Uji8=",
+    "h1:UXiklYAaOCVS4GdN+jYcejSbLEJNEymoTxVICqnVVt0=",
```

Comment:
Nit: This PR should only include control-plane module updates

Reply:
Not sure how this got in there, since I just merged upstream into my fork. I didn't even do a `terraform init`